src/screens/Customers/AddCustomer.tsx

```diff
@@ -78,26 +114,81 @@ function AddCustomer(props: AddCustomerProps) {
-          <Select label={"Select Tariff Plan"} id={'select-tariff-plan'} labelId="select-tariff-plan-label" onChange={handleSelectTariffPlan}>
-            {tariffPlans.map((plan) =>
-              <MenuItem key={plan.id} value={plan.id}>{plan.name}</MenuItem>
-            )}
-          </Select>
-        </FormControl>
-        <Checkbox label={'Meter:'} textField={hasMeter} checkboxOnChange={() => setHasMeter(!hasMeter)} textFieldOnChange={handleMeterInput} />
-        <Checkbox label={'Customer is inactive'} checkboxOnChange={() => setCustomerInactive(!customerInactive)} />
-        <Button label={'Add'} onClick={handleSubmit} />
-      </form>
+      <BaseScrollView>
+        <form noValidate onSubmit={() => false}>
+          <div className={classes.textContainer}>
+            <div className={classes.inline}>
+              <div className={classes.nameInput}>
+                <TextField
+                  label={'Name'}
+                  id={'name'}
+                  placeholder={'e.g. Tom'}
+                  onChange={handleNameInput}
+                  required
+                />
+              </div>
+              <div className={classes.numberInput}>
+                <TextField
+                  label={'Number'}
+                  id={'number'}
+                  placeholder={'e.g. 12'}
+                  type="number"
+                  onChange={handleCustomerNumberInput}
+                  required
+                />
+              </div>
+            </div>
+          </div>
+          <div className={classes.selectContainer}>
+            <FormControl required variant="outlined" className={classes.formControl}>
+              <InputLabel>Meter Type</InputLabel>
+              <Select onChange={handleSelectMeterType}>
+                <MenuItem value={MeterType.ANALOG_METER}>Analog Meter</MenuItem>
+                <MenuItem value={MeterType.SMART_METER}>Smart Meter</MenuItem>
+                <MenuItem value={MeterType.NO_METER}>No Meter</MenuItem>
+              </Select>
+            </FormControl>
+          </div>
+          <div className={classes.textContainer}>
+            <TextField
+              label={'Meter Number'}
+              id={'meter-number'}
+              placeholder={'Input'}
+              type="number"
+              onChange={handleMeterNumberInput}
+              disabled={selectedMeterType === MeterType.NO_METER}
+              required={selectedMeterType != MeterType.NO_METER}
+            />
+          </div>
+          <div className={classes.selectContainer}>
+            <FormControl required variant="outlined" className={classes.formControl}>
+              <InputLabel>Tariff Plan</InputLabel>
+              <Select onChange={handleSelectTariffPlan}>
+                {tariffPlans.map((plan) =>
+                  <MenuItem key={plan.id} value={plan.id}>
+                    <TariffPlanCard tariffPlan={plan} />
+                  </MenuItem>
```

Comment:
This wasn't on the Figma, but the NPO did say they'd like this sorted descending so that the top tariff plan on the select is the one with the most customers, second is the one with the second-most customers, etc